examples/accelerate_configs/deepspeed_zero1.yaml

```diff
@@ -9,7 +9,7 @@ distributed_type: DEEPSPEED
 downcast_bf16: 'no'
 machine_rank: 0
 main_training_function: main
-mixed_precision: 'bf16'
+mixed_precision: 'fp16'
```

Comment:
Just a heads up that I've modified this because the runners (T4) do not support bf16 training

Reply:
actually this is not needed, please check my latest commits and comments